src/test/java/org/kframework/ctests/OuterTest.java

```diff
@@ -73,7 +78,34 @@ public void testOuter() {
         }
     }
 
+	private String findStack() {
+		String path = System.getenv("PATH");
+		String[] paths;
+		if (System.getProperty("os.name").startsWith("Win")) {
+			paths = path.split(";");
+		} else {
+			paths = path.split(":");
+		}
+		for (String ps : paths) {
+			Path p = Paths.get(ps);
+			final PathMatcher filter = p.getFileSystem().getPathMatcher("glob:**stack*");
+			try {
+				try (final Stream<Path> stream = Files.list(p)) {
+					List<Path> l = stream.filter(filter::matches)
+						.collect(Collectors.toList());
+					if (l.size() == 1) {
+						return l.get(0).toString();
+					}
+				}
+			} catch (Exception e){
+				throw new RuntimeException(e);
+			}
+		}
+		throw new RuntimeException("Could not find stack");
+	}
+
     private String process(File f) {
+		String stackPath = findStack();
```

Comment:
When I run it on my system, I get this: `c:\Program Files\Java\jdk-10.0.1\bin\jstack.exe`.
Is there a way to add `kore-parser` directly to `PATH`? Then we don't have to find anything, just run the parser.
On my system it gets generated in `c:\work\kore\src\main\haskell\.stack-work\install\4cb44db3\bin>kore-parser.exe`
but I'm not very familiar with stack and I don't trust that folder name.

Reply:
I think we can probably just use `stack build` or some such.  Instead of globbing for **stack* I could use a regex and search for `.*/stack|.*/stack[.]exe`